src/Routes.js

```diff
@@ -20,6 +22,8 @@ export default () => (
       <Route path='/sign-up' component={SignUp}/>
       <Route path="/task/create" component={Create} />
       <Route path="/task/update" component={Update} />
+      <Route path='/task/list' component={List} />,
```

Comment:
Might be more @kvartborg's area of expertise, but the routes might not be ideally named

Reply:
Do not have a better name so lets keep it for now.